core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -672,11 +672,11 @@ private class MutationWriter {
     public MutationWriter(int numSendThreads) {
       serversMutations = new HashMap<>();
       queued = new HashSet<>();
-      sendThreadPool = context.threadPools().getPoolBuilder(this.getClass().getName())
+      sendThreadPool = context.threadPools().getPoolBuilder("tserver.batch.writer.send.pool")
```

Comment:
`tserver` might be confusing, could probably just get away with `batch.writer...`. Also, probably don't need  `pool` in the name as I think it's already being added by the prefix

Reply:
removed in 3e2982c36b